webblog/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
```

Comment:
Add webpack configurations for `webpack.common.config.js`, `webpack.dev.config.js` (with hot reload) and `webpack.prod.config.js` (export project in `build` folder).